requirements.txt

```diff
@@ -1,7 +1,7 @@
 beautifulsoup4
 colorama==0.4.6
 openai==0.27.2
-playsound==1.3.0
```

Comment:
Why this downgrading?

Reply:
Version 1.3.0 returns an [error during installation](https://stackoverflow.com/questions/68826091/the-specified-device-is-not-open-or-is-not-recognized-by-mci). This change has already been made in a recent [commit](https://github.com/Torantulino/Auto-GPT/commit/3f106963a8b461a52ec1d9acb3a3273cac3ad6d7). Downgrading to playsound 1.2.2 appears to resolve this issue.